template/build/webpack.base.conf.js

```diff
@@ -8,7 +8,7 @@ function resolve (dir) {
   return path.join(__dirname, '..', dir)
 }
 
-const createLintingRule = () => ({
+{{#lint}}const createLintingRule = () => ({
```

Comment:
There's a call to that function further below in the rules section, that should be wrapped in the same handlebars check.

Reply:
It's already wrapped

Reply:
Right, my mistake. Thanks.